### ISSUE TITLE
Document org-wide process for offtopic troll issue response

### ISSUE THREAD

**Issue by sftim** [kubernetes/org#2363]
### Organization or repository
[`kubernetes/org`](https://github.com/kubernetes/org)

### Describe the issue
I'd like the README at https://github.com/kubernetes/org#readme to signpost people (me!) to advice about how a maintainer can and should respond if offtopic troll issues come up.
(the response is likely to be to lock the issue, but it's a rare enough concern that we might not remember the details)

/kind feature
/sig contributor-experience
I think

**Comment by justaugustus** [comment]
Opinion (specifically about the one from earlier):
- Lock immediately (maybe also snapshot)
- Flag @kubernetes/owners 
- Org-level ban
- Delete

**Comment by ameukam** [comment]
@sftim One approach would be to add github guidelines in the set of guidelines we have for most of the collaboration platforms: https://github.com/kubernetes/community/tree/master/communication.

**Comment by mrbobbytables** [comment]
Yup - that is the place with all our comm policies / procedures.
